lib/fetch/index.js

```diff
@@ -1807,11 +1807,11 @@ async function httpNetworkFetch (
       async start (controller) {
         fetchParams.controller.controller = controller
       },
-      async pull (controller) {
-        await pullAlgorithm(controller)
+      pull (controller) {
```

Comment:
cc: @KhafraDev

Reply:
the streams spec will wrap this in a promise, so it does nothing